lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -81,12 +83,19 @@ class ScreenshotThumbnails extends Audit {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
     const cachedThumbnails = new Map();
 
-    return artifacts.requestSpeedline(trace).then(speedline => {
+    return Promise.all([
+      artifacts.requestSpeedline(trace),
+      TTFI.audit(artifacts).catch(() => ({})),
```

Comment:
that's fine. I filed #2275 so we can get consistency (and support TTCI as a computed artifact)